packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -1246,8 +1246,10 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 	 */
 	private async getOutboundRoutes(): Promise<string[]> {
 		const outboundRoutes: string[] = [];
+		// Getting this information is a performance optimization that reduces network calls for virtualized datastores
+		const aliasedDataStores = new Set(this.aliasMap.values());
```

Comment:
wondering if it's worth doing this once in the constructor, then add to this set each time we add to the alias map. This way, we don't have to incur an O(n) cost every time we need to call this function, which is every time GC runs.